packages/ui/cypress/tests/multisig-creation.cy.ts

```diff
@@ -5,54 +5,178 @@ import { multisigPage } from '../support/page-objects/multisigPage'
-    newMultisigPage.thresholdInput().type('2')
-    newMultisigPage.nameInput().type('Some Name')
-    newMultisigPage.nextButton().should('contain', 'Next').click()
-    newMultisigPage.nextButton().should('contain', 'Create').click()
-    cy.getTxRequests().then((req) => {
-      const txRequests = Object.values(req)
-      cy.wrap(txRequests.length).should('eq', 1)
-      cy.wrap(txRequests[0].payload.address).should('eq', address1)
-      newMultisigPage.nextButton().should('not.exist')
-      newMultisigPage.creatingLoader().should('exist')
-      cy.approveTx(txRequests[0].id)
-      notifications.notificationWrapper(10000).should('have.length', 1)
-      notifications.loadingNotificationIcon().should('be.visible')
-      notifications.notificationWrapper().should('contain', 'broadcast')
-
-      notifications.successNotificationIcon(30000).should('be.visible')
-      notifications.notificationWrapper().should('contain', 'Tx in block')
-      const expectedMultisigAddress = 'D9b1mkwhCwyRMUQZLyyKPdVkiJfFCuyVuWr3EmYAV6ETXkX'
-      multisigPage.accountHeader(10000).within(() => {
-        accountDisplay.addressLabel().should('contain.text', expectedMultisigAddress.slice(0, 6))
-      })
+  describe('Multisig creation - step 1', () => {
+    beforeEach(() => {
+      topMenuItems.newMultisigButton().click()
+    })
+
+    it('Should not allow to click next => when creating a multisig with less than 2 Signatories selected', () => {
+      typeAndAdd(address1)
+
+      newMultisigPage.nextButton().should('be.disabled')
+    })
+
+    it('Should add signatory with no name given', () => {
+      typeAndAdd(randomSignatory1)
+
+      newMultisigPage.step1.accountNameInput().should('have.value', '')
+    })
+
+    it('Shows the error message => when signatory address is invalid', () => {
+      typeAndAdd(randomSignatory1.slice(0, 10))
+
+      newMultisigPage.step1
+        .invalidAddressSelection()
+        .should('be.visible')
+        .and('contain', 'Invalid address')
+    })
+
+    it('should show the warning => when signatories address does NOT belong to your account', () => {
+      typeAndAdd(randomSignatory1)
+      typeAndAdd(randomSignatory2)
+
+      newMultisigPage.step1
+        .createMutlisigError()
+        .should('be.visible')
+        .and('contain', 'At least one of your own accounts must be a signatory.')
+    })
+  })
+
+  describe('Multisig creation - step 2', () => {
+    beforeEach(() => {
+      topMenuItems.newMultisigButton().click()
+    })
+
+    it('Should show the warning => when threshold is too high or low', () => {
+      typeAndAdd(address1)
+      typeAndAdd(address2)
+      newMultisigPage.nextButton().should('contain', 'Next').click()
+      const warningMessage = 'Threshold must be between'
+
+      // When is too high
+      newMultisigPage.step2.thresholdInput().type('3')
+      newMultisigPage.step2.thresholdWarning().should('be.visible').and('contain', warningMessage)
+      // When is too low
```

Comment:
please check the "next" button status each time, that it's disabled, then enabled with the "2"